optimum/onnxruntime/modeling_seq2seq.py

```diff
@@ -1076,13 +1083,28 @@ def _from_pretrained(
         if model_save_dir is None:
             model_save_dir = new_model_save_dir
 
+        generation_config = None
+        try:
+            generation_config = GenerationConfig.from_pretrained(
+                model_id,
+                cache_dir=cache_dir,
+                force_download=force_download,
+                local_files_only=local_files_only,
+                use_auth_token=use_auth_token,
+                revision=revision,
+                subfolder=subfolder,
+            )
+        except OSError:
+            logger.info("Generation config file not found, using a generation config created from the model config.")
```

Comment:
```suggestion
            logger.warning("Generation config file not found, using a generation config created from the model config.")
```

Reply:
I would prefer to leave as is as it follows https://github.com/huggingface/transformers/blob/375801d5e65457d2ef86e6b269703c5f9ad67252/src/transformers/modeling_utils.py#L2520-L2522